src/CGRANode.cpp

```diff
@@ -30,6 +30,7 @@ CGRANode::CGRANode(int t_id, int t_x, int t_y) {
   m_canStore = false;
   m_canLoad = false;
   m_supportComplex = false;
+  m_supportPathDim = false;
```

Comment:
Within one tile, its name might not need `Dim`, right? I guess `Dim` is probably indicating number of tiles supporting path merging? If that's the case, what about `m_supportPathMerging`?